pkg/user/handler.go

```diff
@@ -363,6 +383,9 @@ func (h Handler) SignOut(c *gin.Context) {
 	//	200:
 	//	401: Error
 	//	415: Error
+
+	// TODO: cookies should be cleared no matter what...
```

Comment:
Can we do this correctly now?

Reply:
The reason for this comment is that the endpoint is authenticated which is because if we need the user id to delete its associated tokens from Redis. So if the token is already expired we're not clearing the cookies. I'm not sure what the best way to go about this is.